src/msg.c

```diff
@@ -578,7 +518,7 @@ static int8_t msg_auth(struct session *session,
 	/* Set UUID & token: Used at property_changed */
 	session->uuid = l_strdup(uuid);
 	session->token = l_strdup(token);
-	proto_sock = l_io_get_fd(session->proto_channel);
+
 	/* Set Id */
 	mydevice = l_queue_find(device_id_list, device_uuid_cmp, session->uuid);
```

Comment:
You must verify if mydevice is NULL and avoid access it

Reply:
Ok